tests/tool/Integration.Tests/IntegrationScenarios/WpfSample/csharp/Upgraded/BeanTraderClient/BeanTraderClient.csproj

```diff
@@ -33,7 +33,12 @@
   <ItemGroup>
     <PackageReference Include="Castle.Windsor" Version="4.1.1" />
     <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.3" />
+    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
     <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure" Version="3.3.7" />
+    <PackageReference Include="System.Net.Http" Version="4.3.0" />
+    <PackageReference Include="System.ServiceModel.Duplex" Version="4.8.1" />
```

Comment:
Not sure why these had been removed before. I can't see anything that would have transitively brought them in. Only thing I can think of is when we were doing it before it was still against the previous target framework and may have been deemed unnecessary due to that.

Reply:
These are all brought in by Microsoft.Windows.Compatibility. Aren't we supposed to add a reference to that since this is has a -windows TFM? After that happens, these would be transitive.

Reply:
Hmm not sure why it's not showing up (I mean the Microsoft.Windows.Compat doesn't seem to)